internal/configuration/config.go

```diff
@@ -66,8 +64,15 @@ type WavefrontSinkConfig struct {
 	ClusterName string `yaml:"-"`
 }
 
+type CollectionSourceConfig struct {
+	Interval string `yaml:"interval"`
```

Comment:
If these properties are not provided, we should use the global default `CollectionInterval` (see comment above).